js/start.js

```diff
@@ -417,6 +422,29 @@ var loadProfile = function(landingRoute, onboarded) {
               });
 
               $('#sideBar').html(app.chatVw.render().el);
+              startUpLoadingModal.remove();
+
+              // setting up a loadingModal instance that can be used app-wide as necessary.
+              app.loadingModal = new LoadingModal({
+                dismissOnOverlayClick: false,
+                dismissOnEscPress: false,
+                showCloseButton: false
+              }).render();
+              app.loadingModal.__origRemove = app.loadingModal.remove;
+              app.loadingModal.remove = () => {
+                throw new Error('This instance of the loadingModal is globally shared ' +
+                  'and should not be removed. When you are done with it, please close it.');
+              };
+
+              // setting up a simpleMessageModal instance that can be used app-wide as necessary.
+              app.simpleMessageModal = new SimpleMessageModal({
+                removeOnClose: false
+              });
+              app.simpleMessageModal.__origRemove = app.simpleMessageModal.remove;
+              app.simpleMessageModal.remove = () => {
+                throw new Error('This instance of the simpleMessageModal is globally shared ' +
+                  'and should not be removed. When you are done with it, please close it.');
+              };              
 
               app.router = new router({userModel: user, userProfile: userProfile, socketView: newSocketView});
 
```

Comment:
great catches! I've fixed them.

Also, by default I've made the LoadingModal also get those options preventing it from being closable, since I don't see a scenario where we'd want the user to close it out.

A cool improvement would be to make a 'closeable' option that controls those three, which would simplify the syntax, but I think for now it's ok.